### PR TITLE
Add vSphere v20.0.0-alpha1 release

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/roadmap/issues/456

### CODE REVIEW

File: vsphere/releaseNotesTransformer.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: giantswarm.io/v1
+kind: releaseNotesURLAnnotationTransformer
+metadata:
+  name: kvmReleaseNotesURLAnnotationTransformer
```
Comment (pipo02mix) [inline vsphere/releaseNotesTransformer.yaml:4]: is this wrong?
```suggestion
  name: vSphereReleaseNotesURLAnnotationTransformer
```

File: vsphere/releaseNotesTransformer.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+# provider annotationKey
+argsOneLiner: kvm giantswarm.io/release-notes
```
Comment (pipo02mix) [inline vsphere/releaseNotesTransformer.yaml:6]:
```suggestion
argsOneLiner: vSphere giantswarm.io/release-notes
```

**Review by pipo02mix — APPROVED** [review]
LGTM just a comment

**Comment by kopiczko** [comment]
Not sure if this shouldn't be v20.0.0-alpha now: https://gigantic.slack.com/archives/C04TGHDEF/p1635931113012400

File: vsphere/releaseNotesTransformer.yaml (hunk 2 above)
Comment (tfussell) [reply vsphere/releaseNotesTransformer.yaml]: Good call. Fixed it.

**Comment by tfussell** [comment]
> Not sure if this shouldn't be v20.0.0-alpha now: https://gigantic.slack.com/archives/C04TGHDEF/p1635931113012400

I renamed it.

**Comment by tfussell** [comment]
There's probably more to do here, but I'm going to go ahead and merge it since it's not deployed anywhere. We'll improve it when we go back to CAPV.

**Pull request merged**